book/src/commands.md

```diff
@@ -3,3 +3,7 @@
 Command mode can be activated by pressing `:`. The built-in commands are:
 
 {{#include ./generated/typable-cmd.md}}
+
+There also static commands that can be bound to keys and they take no arguments:
```

Comment:
```suggestion
There are also static commands that take no arguments and can be bound to keys. Static commands can also be executed from the command picker (`<space>?`). The built-in static commands are:
```

Let's also mention the command picker (`<space>?`) here since that's the best way of discovering commands while in the editor